### PR TITLE
Download ics file for export academy events

### PR DESCRIPTION
**Pull request by bsabhilash**

### CODE REVIEW

**Review by lewis-coulson-dit — APPROVED** [review]
LGTM

File: export_academy/views.py [hunk 1]
```diff
@@ -103,3 +110,39 @@ def get_context_data(self, **kwargs):
...
+        meeting['UID'] = uuid4()
+
+        description = f'{event.name}\n\n{event.description}{calender_content()}'
+        meeting.add('DESCRIPTION', description)
+
+        file_name = get_valid_filename(event.name)
+        alarm = Alarm()
+        alarm.add('action', 'DISPLAY')
+
+        alert_time = timedelta(minutes=-15)
```
Comment (timothyPatterson) [inline export_academy/views.py:140]: Should this be something like:
``alert_time = event_time - timedelta(minutes=15)``\
Just a question - difficult to tell from docs.

Comment (bsabhilash) [reply export_academy/views.py:140]: No. As per the RFC for iCalendar, we need to give relative time, and client will pick the event start time from event

Comment (timothyPatterson) [reply export_academy/views.py:140]: Perfect thanks

**Pull request merged**